env/pom.xml

```diff
@@ -24,18 +24,15 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>commons-io</groupId>
-            <artifactId>commons-io</artifactId>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
+            <groupId>com.sap.cloud.security.xsuaa</groupId>
```

Comment:
pls add a comment that token-client and http-client are only required, in case of multiple xsuaa service bindings.